src/styles/style.css

```diff
@@ -38,78 +39,85 @@ body {
 
 ul {
   list-style-position: inside;
-  margin-top: 0.65rem;
+  margin-top: var(--gcds-spacing-300);
 }
 
 ul li {
-  margin-top: 0.433rem;
+  margin-top: var(--gcds-spacing-200);
 }
 
 /*
  * TYPOGRAPHY
  */
 
+h1,
+h2,
+h3,
+h4,
+h5,
```

Comment:
According to our typography page, h5 and h6 share the same font family as the body.

Reply:
Oh good catch, thank you! Updated it.